app/models/constant.rb

```diff
@@ -0,0 +1,9 @@
+class Constant < ActiveRecord::Base
+  validates_presence_of :name, :value
+  validates_uniqueness_of :name
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.